build.gradle

```diff
@@ -65,4 +65,40 @@ dependencies {
 
 test {
 	useJUnitPlatform()
+}
+
+
+apply plugin: "java"
+
+repositories {
+	jcenter()
+}
+
+configurations {
+	ktlint
+}
+
+dependencies {
+	ktlint "com.github.shyiko:ktlint:0.27.0"
+	// additional 3rd party ruleset(s) can be specified here
+	// just add them to the classpath (ktlint 'groupId:artifactId:version') and
+	// ktlint will pick them up
+}
+
+task ktlint(type: JavaExec, group: "verification") {
+	description = "Check Kotlin code style."
+	main = "com.github.shyiko.ktlint.Main"
+	classpath = configurations.ktlint
+	args "src/**/*.kt"
+	// to generate report in checkstyle format prepend following args:
+	// "--reporter=plain", "--reporter=checkstyle,output=${buildDir}/ktlint.xml"
+	// see https://github.com/shyiko/ktlint#usage for more
+}
+check.dependsOn ktlint
+
+task ktlintFormat(type: JavaExec, group: "formatting") {
+	description = "Fix Kotlin code style deviations."
+	main = "com.github.shyiko.ktlint.Main"
+	classpath = configurations.ktlint
+	args "-F", "src/**/*.kt"
```

Comment:
This is just a copy-paste from https://ktlint.github.io/.  Should I merge this stuff into the existing sections of the file?

Reply:
That's a great question. I've been thinking about this a bit as I iteratively add to the gradle file, and how separating everything out so that you have the minimal number of sections ends up (a) making it harder to see which pieces are connected to what and (b) making it fiddlier when needing to make a number of related changes across multiple larger sections.

Check this out: https://docs.gradle.org/current/userguide/plugins.html#sec:script_plugins

What do you think of potentially separating out very specific large groups of related declarations into separate files? This might only really be ktlint for now, but it's possible some more groups like this will arise as we flesh out the build file.

I could also see just using comments and keeping specific related groups of declarations in separate sections as it is here.